package.json

```
@@ -40,7 +40,7 @@
     "snyk": "^1.364.0"
   },
   "peerDependencies": {
-    "aws-sdk": "^2.716.0"
+    "aws-sdk": "^2.743.0"
```

```suggestion
    "aws-sdk": ">=2.743.0"
```
This can be more flexible to avoid the need for updating again.